t/16-utils-runcmd.t

```diff
@@ -57,20 +55,24 @@ subtest 'make git commit (error handling)' => sub {
 
 # setup mocking
 my @executed_commands;
-my $utils_mock = Test::MockModule->new('OpenQA::Git');
 my %mock_return_value = (
     status => 1,
     stderr => undef,
 );
-$utils_mock->redefine(
-    run_cmd_with_log_return_error => sub {
-        my ($cmd) = @_;
-        push(@executed_commands, $cmd);
-        return \%mock_return_value;
-    });
 
-subtest 'git commands with mocked run_cmd_with_log_return_error' => sub {
+sub _run_cmd_mock{
+  my ($cmd) = @_;
```

Comment:
Use signatures in new code:
```suggestion
sub _run_cmd_mock ($cmd) {
```
(Maybe you'll need to add `, @` as the might be more parameters passed.)